CMakeLists.txt

```diff
@@ -0,0 +1,51 @@
+# Select system name Generic and change it to proper by target architecture settings files
```

Comment:
@IBeRyUS Can we please use capital letters for directory names as described in the Build System spec?
environment -> Environment
build -> Out
hello->Hello
src->Source

Reply:
I'm challenging UpperCase letters for first character of folder names

Reply:
How to decide then :) A polling?